clang/lib/Analysis/LiveVariables.cpp

```diff
@@ -90,8 +90,8 @@ namespace {
     if (A.isEmpty())
       return B;
 
-    for (typename SET::iterator it = B.begin(), ei = B.end(); it != ei; ++it) {
-      A = A.add(*it);
+    for (const auto &element : B) {
```

Comment:
We have pointers in these sets. Maybe iterating by value would be better to avoid double indirection?